vmm/src/config.rs

```diff
@@ -1915,6 +1920,20 @@ impl VmConfig {
                 return Err(ValidationError::UserDevicesRequireSharedMemory);
             }
         }
+
+        if let Some(numa) = &self.numa {
+            let mut used_numa_node_memory_zones = Vec::new();
+            for numa_node in numa.iter() {
+                for memory_zone in numa_node.memory_zones.clone().unwrap().iter() {
+                    if !used_numa_node_memory_zones.contains(memory_zone) {
+                        used_numa_node_memory_zones.push(memory_zone.to_string());
+                    } else {
+                        return Err(ValidationError::MemoryZoneReused(memory_zone.to_string()));
```

Comment:
This is nice, but could your also give the conficting node ids to make it easier for the user to find their conflict?

Reply:
Done. Yes that makes even more sense :))